synapse/lib/modelrev.py

```diff
@@ -1205,3 +1206,761 @@ async def _propArrayToForm(self, layers, propfull, formname):
         }
         '''
         await self.runStorm(storm, opts=opts)
+
+class ModelMigration_0_2_31:
```

Comment:
Is this supposed to be a `Base` subclass?

Reply:
Nope. I didn't think it needed all the weight or any of the utility of Base.